share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -268,10 +268,13 @@ function(ExternalZephyrProject_Add)
       )
     endif()
 
-     # Check for sysbuild related configuration fragments.
-     # The contents of these are appended to the image existing configuration
-     # when user is not specifying custom fragments.
-    if(NOT "${CONF_FILE_BUILD_TYPE}" STREQUAL "")
+    # Check for sysbuild related configuration fragments.
+    # The contents of these are appended to the image existing configuration
+    # when user is not specifying custom fragments. Note: FILE_SUFFIX will take priority over
+    # a prj_<X> build type if both are used
+    if(DEFINED FILE_SUFFIX AND EXISTS "${sysbuild_image_conf_dir}/${ZBUILD_APPLICATION}_${FILE_SUFFIX}.conf")
+      set(sysbuild_image_conf_fragment ${sysbuild_image_conf_dir}/${ZBUILD_APPLICATION}_${FILE_SUFFIX}.conf)
```

Comment:
By applying a `suffix` in both sysbuild and the image itself, you may end up using `image_suffix_suffix.conf`. Why is the sysbuild change needed?

Reply:
This is if someone has `<app>/sysbuild/mcuboot_<X>.conf` rather than `<app>/sysbuild/mcuboot/prj_<X>.conf` so is handled in sysbuild

Reply:
Fixes